pokedex/pokemon.lua

```diff
@@ -431,7 +445,9 @@ function M.get_evolution_points(pokemon)
 		while next(evolutions) ~= nil do
 			table.remove(evolutions)
 			current = pokedex.get_evolved_from(current)
-			evolution_points = evolution_points + pokedex.evolve_points(current)
+			if current then
```

Comment:
current can now be nil (previously was MissingNo) so I protected against it just in case.